src/Forms/Shared/Samples/GeometryEngine/Buffer/Buffer.xaml

```diff
@@ -2,21 +2,56 @@
 <ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
-             xmlns:local="clr-namespace:ArcGISRuntime.Samples.Buffer"
              x:Class="ArcGISRuntime.Samples.Buffer.Buffer">
     <Grid>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-        </Grid.RowDefinitions>
-        <StackLayout Grid.Row="0" 
-                     Orientation="Vertical">
-            <Label x:Name="BufferInstructionsLabel" 
-                   Text="Specify the buffer in miles and the tap on the map:"/>
-            <Entry x:Name="BufferDistanceMilesEntry" 
-                   Text="10"/>
-        </StackLayout>
-        <esriUI:MapView x:Name="MyMapView" 
-                        Grid.Row="1"/>
+        <esriUI:MapView x:Name="MyMapView"/>
+        <Frame BackgroundColor="DarkGray" Opacity="0.8"
```

Comment:
Although this follows the pattern on UWP and WPF, it can be difficult to use on the phone (especially simulators because of how they implement multitouch). 

I recommend moving this above the mapview so that they don't overlap. It will be a bit uglier on most platforms, but should be easier to use.